### PR TITLE
[implementations] Restructure the description of the reference implementations

### PR DESCRIPTION
**Pull request by jgbarah**
I've tried to make a more clear and comprehensive description of the reference implementations. Currently, we're working on the implementations based on Pandas data frames, but we're also preparing the path for pure Python implementations.

### CODE REVIEW

**Comment by jgbarah** [comment]
Any comment???

File: implementations/README.md [hunk 1]
```diff
@@ -1,9 +1,44 @@
-# Examples of implementations for metrics
+# Reference implementations
 
-Notebooks, built on top of data collected by Perceval,
-for illustrating how to compute different metrics of the Evolution Working Group. The notebooks will be explanatory in regard to the analysis performed. They should help the user understand how to calculate these metrics, as well as to understand how the metric is defined.
+This directory provides reference implementations for the metrics defined by the
+woeking group. All these reference implementations use data produced by
```
Comment (germonprez) [inline implementations/README.md]: working group

File: implementations/README.md [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
 
-The contents of this directory are the notebooks(present in [notebooks_df](./notebooks_df/), as well as some scripts in the [code_df](./code_df/) package. The "_df" suffix is to show that these scripts and notebooks make use of the Pandas library. `code_df` contains the following modules:
+## List of reference implementations
+
+* **Code Changes ([module](./code_df/code_changes_git.py) • [notebook](./notebooks_df/code_changes_git.ipynb) • [description](https://github.com/chaoss/wg-evolution/blob/master/metrics/Code_Changes.md))**  
+    This metric is computed for the data in the file `git-commits.json`.
+
+## Contents of this directory
+
+As the framework for producing the implemenations we use Python Jupyter notebooks,
```
Comment (germonprez) [inline implementations/README.md]: implementations

**Comment by GeorgLink** [comment]
@germonprez I see you found the same typos :D I had started my review during the call and never submitted it :-o

**Comment by jgbarah** [comment]
Thanks a lot for your comments. Please, check the current version, with the problems hopefully fixed.

**Comment by jgbarah** [comment]
@germonprez @GeorgLink could you please check if my changes are good enough to accept the pull request?

**Comment by germonprez** [comment]
Thanks @jgbarah

**Pull request merged**